package/MDAnalysis/core/AtomGroup.py

```diff
@@ -3336,6 +3378,7 @@ def __init__(self, name, id, atoms, resnum=None):
         ##if not Residue._cache.has_key(name):
         ##    Residue._cache[name] = dict([(a.name, i) for i, a in enumerate(self._atoms)])
 
+    @deprecate(message=_FIFTEEN_DEPRECATION)
```

Comment:
Yeah rather leave a note in the doc string so people can look into it if it fails.